frontend/rolecall/src/app/api/piece_api.service.ts

```diff
@@ -288,4 +288,8 @@ export class PieceApi {
       }
     });
   }
+
+  static uuidFromRaw(id: number) {
```

Comment:
No need to use static since it's a basic util, just pull it out of the class and export it as a simple function. Will be easier to refactor later when we have a bunch of them and they need to extract to a separate file.

Reply:
You are right that there is no need right now. However, the id -> uuid transformation could easily be much more involved. If that case it is also likely that different classes would have different logic for going from a database id to a frontend uuid. I though it was a good idea to make every class have a static function that you can call every time you need to generate a uuid from a database id.

Reply:
Removed.